app/src/main/res/layout/activity_main.xml

```diff
@@ -181,6 +181,13 @@
                 mapbox:icon="@drawable/img_layer_icon" />
 
         </androidx.cardview.widget.CardView>
```

Comment:
Please, add a small commentary to tag this section as the offline mode components